### PR TITLE
Unethical corp/project assignment

### PR DESCRIPTION
**Pull request by thuang814**
Features:
-completed the project assignment page with a question as title and radio buttons for answer choices
-added completed boolean object to pages object in simulation window
       -if you havent completed the question the next button does work
       -you have to have visited response one by clicking a answer in order for next button to work when you go back
-addressed some pr comments

### CODE REVIEW

File: frontend/src/pages/results.js [hunk 1]
```diff
@@ -41,15 +41,15 @@ function Results({pages, setPages, activePage, setActivePage}) {
...
           <Button variant="contained" disableElevation onClick={goToStakeholders}>Back</Button>
         </Grid>
-        <Grid item style={{ marginRight: "2rem", marginRight: "0rem", marginTop: "-3rem" }}>
+        <Grid item style={{ marginRight: "0rem", marginTop: "-3rem" }}>
           <Button variant="contained" disableElevation color="primary" onClick={goToFeedback}>Next</Button>
         </Grid>
       </Grid>
       <Grid container spacing={2}>
-        <Grid item lg={12}>
+        <Grid item XS={48} sm={24} lg={12}>
```
Comment (powellsl99) [inline frontend/src/pages/results.js]: To the best of my knowledge this isn't how you want to use grid sizing. It goes from 1-12. So ={12} takes up the whole grid length, ={6} is half, etc

File: frontend/src/pages/simulationWindow.js [hunk 2]
```diff
@@ -16,14 +15,14 @@ function SimulationWindow() {
...
   const [pages, setPages] = useState({
-    introduction: {visited: true, pageNumber: 1, html: (<Introduction/>)},
-    projectAssignment: {visited: false, pageNumber: 2, html: (<ProjectAssignment/>)},
-    responseOne: {visited: false, pageNumber: 3, html: (<ResponseOne/>)},
-    gatheredInformation: {visited: false, pageNumber: 4, html: (<GatheredInformation/>)},
-    stakeholders: {visited: false, pageNumber: 5, html: (<Stakeholders/>)},
-    results: {visited: false, pageNumber: 6, html: (<Results/>)},
-    feedback: {visited: false, pageNumber: 7, html: (<Feedback/>)},
-    response: {visited: false, pageNumber: 8, html: (<Response/>)}
+    introduction: {visited: true, completed: true, pageNumber: 1, html: (<Introduction/>)},
```
Comment (powellsl99) [inline frontend/src/pages/simulationWindow.js:18]: I think this is supposed to be dynamic right? Like there can be multiple of certain components and the order can change. Could be wrong, but it would be worth asking the customer if you are unsure (unless this is just hardcoded for demo purposes)

File: frontend/src/pages/response.js [hunk 3]
```diff
@@ -46,6 +45,9 @@ function Response({pages, setPages, activePage, setActivePage}) {
             Some Response.
           </TextTypography>
         </Grid>
+        <Grid item lg={12}>
```
Comment (powellsl99) [inline frontend/src/pages/response.js]: why is this here?

File: frontend/src/pages/chartTest.js [hunk 4]
```diff
@@ -1,107 +1,157 @@
...
+      issue2: 4,
+      issue3: 2,
+    },
+    Song: {
+      issue1: 0,
+      issue2: 2,
+      issue3: 3,
+    },
+  };
+  function Total_EmphasisF(scenario_scoring_matrix) {
```
Comment (powellsl99) [inline frontend/src/pages/chartTest.js]: Is this consistent of how everything across the app will be named? The mix of camel case and snake case seems a but off to me. I think picking camel case (totalEmphasisFunction) or snake case (total_emphasis_function) would be good (generally react projects use camel but its up to you)

File: frontend/src/pages/chartTest.js [hunk 5]
```diff
@@ -1,107 +1,157 @@
...
-        }
+      Student_Emphasis = {
+        issue1: testInput1,
+        issue2: testInput2,
+        issue3: testInput3,
+      };
     }
+    return Student_Emphasis;
+  }
+  let Student_Emphasis = Student_EmphasisF(scenario_scoring_matrix);
```
Comment (powellsl99) [inline frontend/src/pages/chartTest.js]: same here, normally variables don't start upper cased, that's generally reserved for components. Not a huge deal though if this is what you prefer

File: frontend/src/pages/chartTest.js [hunk 6]
```diff
@@ -1,107 +1,157 @@
...
-            for (let issue in scenario_scoring_matrix[conversant]) {
-                Student_Emphasis[issue] = (Student_Emphasis[issue] === undefined) ? scenario_scoring_matrix[conversant][issue] : scenario_scoring_matrix[conversant][issue] + Student_Emphasis[issue]
-            }
+      for (let conversant in scenario_scoring_matrix) {
+        for (let issue in scenario_scoring_matrix[conversant]) {
+          Student_Emphasis[issue] =
+            Student_Emphasis[issue] === undefined
+              ? scenario_scoring_matrix[conversant][issue]
+              : scenario_scoring_matrix[conversant][issue] +
+                Student_Emphasis[issue];
```
Comment (powellsl99) [inline frontend/src/pages/chartTest.js]: This might be a good use for the null coalescing operator since the ternary is a bit messy looking here.
eg. Student_Emphasis[issue] = scenario_scoring_matrix[conversant][issue] + (Student_Emphasis[issue] ?? 0)
I think that will work. You can just keep it as is if not.

**Review by powellsl99 — CHANGES_REQUESTED** [review]
Looks great! Just a few small suggestions or questions!

File: frontend/src/pages/simulationWindow.js (hunk 2 above)
Comment (thuang814) [reply frontend/src/pages/simulationWindow.js:18]: This is hard coded for demo purposes
